lessons/sql/lesson.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: page
+title: 'Databases and SQL'
+visible: true
+---
+
+## Introduction to Databases and SQL ##
+
+- **Authors**: Ian Dennis Miller
+- **Research field**: Social Psychology - Social Complexity
+- **Lesson topic**: Databases and SQL
+- **Lesson content URL**: <https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/sql>
+
+## Presentation
+
+http://iandennismiller.github.io/databases-sql
```

Comment:
People can try it out after the fact. It went by too quickly for anyone to actually give it a try, which is fine. Thanks for the session :)